packages/shared/provideLocal/index.ts

```diff
@@ -1,6 +1,9 @@
+import type { InjectionKey } from 'vue'
 import { getCurrentInstance, provide } from 'vue'
 import { localProvidedStateMap } from './map'
 
+export type ProvideLocalReturn = void
```

Comment:
do we gain anything in these cases where the return type is `void`?

i wonder if this is a waste until we have a non-void return type

Reply:
I thought its better for docs, because we have always a consitient return type.